aii-ks/src/main/pan/quattor/aii/ks/config.pan

```diff
@@ -297,7 +297,7 @@ variable  AII_OSINSTALL_BOOTPROTO ?= 'dhcp';
 # Options for authentication
 # defaults to using shadow passwords and MD5 hashing
 #
-variable AII_OSINSTALL_OPTION_AUTH ?= list ("enableshadow", "enablemd5");
+variable AII_OSINSTALL_OPTION_AUTH ?= list ("enableshadow", "passalgo=sha512");
```

Comment:
ugh - not relevant to this PR but the fact this needs updating in multiple places is reason why the way AII uses global variables like this is bad practice. :( Wish we could eliminate it.